e2e_tests/integration/connect-form.spec.ts

```diff
@@ -99,6 +99,10 @@ describe('Connect form', () => {
         'contain',
         `Automatic retry using the "${schemeWithEncryptionFlag('bolt')}"`
       )
+      getFirstFrameStatusbar().should(
+        'not.contain',
+        `Automatic retry using the "${schemeWithEncryptionFlag('bolt')}"`
+      )
```

Comment:
It's an extra wait, the test was flaky in teamcity when connecting is slow. We could probably alias the request and wait for it instead of this text to disappear, this was an easy fix